package/fingerterm/package

```diff
@@ -5,20 +5,20 @@
 pkgnames=(fingerterm)
 pkgdesc="Terminal emulator with an on-screen touch keyboard"
 url=https://github.com/dixonary/fingerterm-reMarkable
-pkgver=1.3.5-11
-timestamp=2017-12-08T15:40Z
+pkgver=1.3.5-12
+timestamp=2020-10-27T12:02Z
 section="admin"
 maintainer="Mattéo Delabre <spam@delab.re>"
 license=GPL-2.0-or-later
 
 image=qt:v1.1
```

Comment:
```suggestion
image=qt:v1.2.2
```

Might upgrade this as well while at it. (Tested with both versions (and each on both models), so I'll can approve it no matter what version gets used.)

Reply:
Good point. Thanks for testing!